testsuite/MDAnalysisTests/coordinates/test_reader_api.py

```diff
@@ -131,56 +132,71 @@ def _check_slice(self, start, stop, step):
         ref = self.reference[start:stop:step]
 
         assert_equal(res, ref)
-    
+
     def test_slices(self):
         for start, stop, step in [
-                (None, None, None),  # blank slice
-                (None, 5, None),  # set end point
-                (2, None, None),  # set start point
-                (2, 5, None),  # start & end
-                (None, None, 2),  # set skip
-                (None, None, -1),  # backwards skip
-                (0, 10, 1),
-                (0, 10, 2),
-                (None, 20, None),  # end beyond real end
-                (None, 20, 2),  # with skip
-                (0, 5, 2),
-                (5, None, -1),
-                (None, 5, -1),
+            (None, None, None),  # blank slice
+            (None, 5, None),  # set end point
+            (2, None, None),  # set start point
+            (2, 5, None),  # start & end
+            (None, None, 2),  # set skip
+            (None, None, -1),  # backwards skip
+            (0, 10, 1),
+            (0, 10, 2),
+            (None, 20, None),  # end beyond real end
+            (None, 20, 2),  # with skip
+            (0, 5, 2),
+            (5, None, -1),
+            (None, 5, -1),
+            (100, 10, 1),
+            (-10, None, 1),
+            (100, None, -1),  # beyond real end
+            (100, 5, -20)
         ]:
             yield self._check_slice, start, stop, step
 
     def test_slice_IE_1a(self):
```

Comment:
This test should get removed as it no longer raises an Error, and the values for the testing (`[5:1:1]`) should get moved into the test generator above.  The same then applies for the other tests that now work